testpmd/hooks/roles/apply-cnf-cert-config/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- block:
+
+    - name: install
+      include_tasks: install.yml
+
+    - name: post-run
+      include_tasks: post-run.yml
+
+  when: do_cnf_cert|default(false)|bool
```

Comment:
This seems a bit strange to read for me, because in my mind `do_cnf_cert` is a variable that is expected for the hook (example-cnf-config) if that variable is not set to true then this role `apply-cnf-cert-config` should not be called.

When something is calling this role is because it's expected to be executed, that's why to me feels like the `do_cn_cert` is not necessary in here and I think it can be flattened out. 

Anyway, we are only using this as a placeholder because we use the tasks (install, post-run) in a very specific way, so there's no rush to change this one :smile_cat:

Reply:
Yes, in fact, this main.yml file should not be called as we have included explicitly the references to install and post-run tasks, but I just wanted to follow the same structure we are using in d-o-a-a [here](https://github.com/redhat-cip/dci-openshift-app-agent/blob/master/roles/cnf-cert/tasks/main.yml). But it can be safely removed in the future if required :)